src/ServerConfigParams.ts

```diff
@@ -210,8 +210,7 @@ export function parseServerConfig(args: string[], env: any): any {
 
     const config = {
         ...configFile,
-        ...argv,
-        logLevel: configFile.logLevel || env.LOG_LEVEL
+        ...argv
```

Comment:
@ironFe93  I don't think we needed the line just removed because the `logLevel` parameter of env and config should be taken into account at lines 190 and 180 respectively.
Furthermore, if you think we still need it, we should consider adding it as `logLevel: configFile.logLevel ?? env.LOG_LEVEL` otherwise if `configFile` try to set `logLevel` to `0` and it's not defined in the `env`, we'll end up having the value set with `undefined`

Reply:
I'd suggest a cleaner solution of cascading overrides. Something like:
```
const configFile = ...;
const envOverrides = {
  ...,
  logLevel: env.LOG_LEVEL,
  ...
};
const argOverrides = parseArgs(...);

export const config = {
  ...configFile,
  ...envOverrides,
  ...argOverrides
};
```

Reply:
Yep, my point here is that we don't need to explicitly set the logLevel. It will be set as any other config params.

Reply:
Alright, I'm happy to take this approach.

Reply:
I've created a [task](https://rsklabs.atlassian.net/browse/PP-272) to address the refactor of the configuration builder.